src/uu/df/src/table.rs

```diff
@@ -482,4 +482,34 @@ mod tests {
             "my_device        my_type        4.0Ki        1.0Ki        3.0Ki   25% my_mount        "
         );
     }
+
+    #[test]
```

Comment:
please also add a test in https://github.com/uutils/coreutils/blob/main/tests/by-util/test_df.rs

Reply:
Ok, added in https://github.com/uutils/coreutils/pull/3209/commits/b167d18b48ddf4947b1fb8caa4781856b2b23bc3